Client/src/Pages/AdvancedSettings/index.jsx

```diff
@@ -66,24 +81,57 @@ const AdvancedSettings = ({ isAdmin }) => {
 		warn: 4,
 	};
 
+	const unitItems = [
+		{ _id: "d", name: "days" },
+		{ _id: "h", name: "hours" }
+	];
```

Comment:
Maybe we don't need this array. We can do something like:
const unitItems = Object.keys(unitItemLookup).map((key, index) => ({_id: index, name: key})).

Like that we only have one place to update if we need to add or remove items

Reply:
Sounds good, changed here: https://github.com/bluewave-labs/bluewave-uptime/pull/1020/commits/1526b50fd2044ca2df035c6f0a41eaab0e3c56c2